core/src/main/scala/org/apache/spark/sql/CachedPlanKey.scala

```diff
@@ -0,0 +1,86 @@
+/*
```

Comment:
Yes. Using the CachedKey case class which I have defined in the SnappySession only. Will remove this.